src/styles/index.scss

```diff
@@ -13,14 +13,14 @@
 // Use default CMSDS font-faces
 @import "@cmsgov/design-system/dist/scss/font";
 
-// Utilities: Utility classes for applying specific CSS properties
-@import "@cmsgov/design-system/dist/scss/utilities/index";
-//@import "utilities/index";
-
 // Base: HTML elements (i.e. ds-base, ds-h1, ds-text), typography, links, icons
 @import "@cmsgov/design-system/dist/scss/base/index";
 //@import "base/index";
 
 // Components: UI component styles (i.e. ds-c-button, ds-c-field)
 @import "@cmsgov/design-system/dist/scss/components/index";
-@import "components/index";
\ No newline at end of file
+@import "components/index";
+
+// Utilities: Utility classes for applying specific CSS properties
+@import "@cmsgov/design-system/dist/scss/utilities/index";
+//@import "utilities/index";
```

Comment:
Needs new line here

Reply:
It's just a commented placeholder.

Reply:
No I mean every file should end with a new line. Its a linting rule